dist/images/Dockerfile.fedora

```diff
@@ -24,21 +24,21 @@ ENV PYTHONDONTWRITEBYTECODE yes
 RUN INSTALL_PKGS="git rpm-build dnf-plugins-core" && \
     dnf install --best --refresh -y --setopt=tsflags=nodocs $INSTALL_PKGS
 
-# Clone OVS Source Code.
-ARG OVS_REPO=https://github.com/openvswitch/ovs.git
-ARG OVS_GITREF=branch-*
```

Comment:
Can we still keeping the old behaviour if users want to specify OVS ? so we can do something like 
```
ARG OVS_REPO=submodule
```
And later on we check `if [ "OVS_REPO" == "submodule ]"` the code you are doing if not the old way.

Reply:
> I can do that, but the question is whether someone will ever use it? maybe you have an example of when you needed a newer ovs version than the one that OVN uses?

When you need a fix at OVS but not a OVN (way less frequent though), so to test that the fix should be backport to the stable branch where OVN is taking OVS from and then we will pass OVS_REPO and OVS_GITREF to that and OVS from submodule will be replaced by it.

Reply:
Good enough